p2p/transport_test.go

```diff
@@ -517,9 +517,9 @@ func TestTransportMultiplexRejectSelf(t *testing.T) {
 	}()
 
 	if err := <-errc; err != nil {
-		if err, ok := err.(ErrRejected); ok {
-			if !err.IsSelf() {
-				t.Errorf("expected to reject self, got: %v", err)
+		if e, ok := err.(ErrRejected); ok {
```

Comment:
Not sure why this change is made here, but think we should keep the scope of this PR just to ABCI++.